src/stores/room-list/algorithms/list-ordering/ImportanceAlgorithm.ts

```diff
@@ -104,7 +104,6 @@ export class ImportanceAlgorithm extends OrderingAlgorithm {
             // Every other sorting type affects the categories, not the whole tag.
             const categorized = this.categorizeRooms(rooms);
             for (const category of Object.keys(categorized)) {
-                if (!isNaN(Number(category))) continue;
```

Comment:
Can we add a test for this? Just a general rooms should be sorted as expected test.